test/phoenix/verified_routes_test.exs

```diff
@@ -553,6 +584,9 @@ defmodule Phoenix.VerifiedRoutesTest do
           end)
 
         assert warnings == ""
+      after
+        :code.purge(__MODULE__.VerifyFalseTrueMatchesFirst)
+        :code.delete(__MODULE__.VerifyFalseTrueMatchesFirst)
```

Comment:
You could also use tags for making this cleaner.

    @tag cleanup: SomeMod

and then on setup you have on_exit that purges it if the cleanup is set.

Reply:
Is there a way to get a list of modules defined by the current process? Pseudo-code

```elixir
setup do
  on_exit(fn -> Enum.each(Process.defined_modules(), fn mod ->
    :code.purge(mod)
    :code.delete(mod)
  end)
end
```

That would also help in some Elixir tests where we found some missing purges recently

Reply:
There isn’t, unfortunately